src/data/community.json

```diff
@@ -97,5 +99,19 @@
                 "external": true
             }
         }
+    },
+    "section-social-media": {
+        "title": "Social Media",
+        "anchor": "social-media",
+        "textblock": ["You can also find the most important information about the Corona-Warn-App, its features and its development on social media."],
+        "socialmedia": [{
+            "title": "Twitter",
+            "textblock": ["The Corona-Warn-App has its own <a href='https://twitter.com/coronawarnapp' target='_blank'>Twitter-Account</a>. On Twitter, we regularly inform you about key figures, updates and new features. You can also participate in polls and discuss with other followers about the Corona-Warn-App.",
+            "Follow the account, become a part of the Twitter community and help us to make the app even more known and better."]
+        },
+        {
+            "title": "Instagram",
+            "textblock": ["At <a href='https://www.instagram.com/rki_fuer_euch/' target='_blank'>RKI für Euch</a> on Instagram we share interesting facts about the Robert Koch Institute, about the development of the pandemic and about the Corona-Warn-App."]
```

Comment:
I suggest to mention that the Instagram account is German language.

Reply:
Thanks @MikeMcC399! Since the content on Twitter is German as well, I've added the sentence "Note that these accounts only publish German content." below the Social Media headline in the latest commit.